django/db/migrations/operations/models.py

```diff
@@ -147,6 +150,8 @@ def reduce(self, operation, in_between, app_label=None):
                     managers=self.managers,
                 ),
             ]
+        elif isinstance(operation, CreateModel):
+            return True
```

Comment:
I'm not sure I can understand what do you mean.

Returning `True` here means that if we have
```
operations = [
    migrations.CreateModel('Person', [("last_name", models.CharField(max_length=255))]),
    migrations.CreateModel('Book', [('author', models.ForeignKey('test_app.Person', models.CASCADE))]),
    migrations.AddField('person', 'first_name', models.CharField(max_length=255)),
]
```
then `CreateModel('Book', ...)` will not prevent `CreateModel('Person', ...)` from trying to reduce following operations. Thus operations will be reduced to 

```
operations = [
    migrations.CreateModel('Person', [("last_name", models.CharField(max_length=255)), 'first_name', models.CharField(max_length=255)]),
    migrations.CreateModel('Book', [('author', models.ForeignKey('test_app.Person', models.CASCADE))]),
]
```

Isn't that valid optimization?